### ISSUE TITLE
Integration: can not use conda

### ISSUE THREAD

**Issue by JuHyung-Son** [warpdotdev/warp#235]
### Describe the bug

```shell
Version
v0.2021.09.21.20.54.stable_00

Platform
mac

Description
after using conda env, env name is included in line
```

### To Reproduce

1. make conda env
2. activate conda env
3. run any command

### Expected behaviour

```shell
there should be no parse error
```

### Screenshots

![image](https://user-images.githubusercontent.com/20140126/135291646-4be6696a-3b27-434b-994b-b62fad74b25f.png)

### Operating System

MacOS

**Comment by alokedesai** [comment]
Thanks for filling! This seems like an issue with typeahead, we'll make sure to get this fixed.

**Comment by elviskahoro** [comment]
Closed: v0.2021.11.09.19.46.stable_00
